### PR TITLE
doc: backport 7.x add source verification docs v1 6910

### PR DESCRIPTION
**Pull request by jmtaylor90**
Ticket: #6910

Signed-off-by: jason taylor <jtfas90@gmail.com>
(cherry picked from commit b5b81648a9fe4e8404a7d2d40ec0019bc639dcef)

Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [x] I have read the contributing guide lines at
   https://docs.suricata.io/en/latest/devguide/contributing/contribution-process.html
- [x] I have signed the Open Information Security Foundation contribution agreement at
   https://suricata.io/about/contribution-agreement/ (note: this is only required once)
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)
- [x] I have created a ticket at
      https://redmine.openinfosecfoundation.org/projects/suricata/issues
      (if applicable)
https://redmine.openinfosecfoundation.org/issues/6910

Link to ticket: https://redmine.openinfosecfoundation.org/issues/
https://redmine.openinfosecfoundation.org/issues/6910
Describe changes:
- backport cherry-pick of #11133 for suricata 7 branch
-
-

### Provide values to any of the below to override the defaults.

- To use an LibHTP, Suricata-Verify or Suricata-Update pull request,
  link to the pull request in the respective `_BRANCH` variable.
- Leave unused overrides blank or remove.

SV_REPO=
SV_BRANCH=
SU_REPO=
SU_BRANCH=
LIBHTP_REPO=
LIBHTP_BRANCH=

### CODE REVIEW

File: doc/userguide/install.rst [hunk 1]
```diff
@@ -16,6 +16,9 @@ Source
 
 Installing from the source distribution files gives the most control over the Suricata installation.
 
+The Suricata source distribution files should be verified before building
+the source, see :doc:`verifying-source-files`.
+
 Basic steps::
 
     tar xzvf suricata-6.0.0.tar.gz
```
Comment (jufajardini) [inline doc/userguide/install.rst:24]: thinking we could replace `6` with `7` here, to be more in theme :P

**Review by jufajardini — CHANGES_REQUESTED** [review]
The commit-check failure seems to be that the commit-hash is wrong in the commit message - 47d6c3a3ab632dca72

File: doc/userguide/install.rst (hunk 1 above)
Comment (jmtaylor90) [reply doc/userguide/install.rst:24]: will update for the next pr

**Comment by jmtaylor90** [comment]
> The commit-check failure seems to be that the commit-hash is wrong in the commit message - [47d6c3a](https://github.com/OISF/suricata/commit/47d6c3a3ab632dca72e60b753afa60521f0ea079)

Interesting, so which commit would I choose when doing the cherry-pick -x? or is that not an option in this case?

**Comment by jufajardini** [comment]
> > The commit-check failure seems to be that the commit-hash is wrong in the commit message - [47d6c3a](https://github.com/OISF/suricata/commit/47d6c3a3ab632dca72e60b753afa60521f0ea079)
> 
> Interesting, so which commit would I choose when doing the cherry-pick -x? or is that not an option in this case?

I think that the one that I indicated in the message should work, as that's the hash of the merged one :)

**Comment by jmtaylor90** [comment]
> > > The commit-check failure seems to be that the commit-hash is wrong in the commit message - [47d6c3a](https://github.com/OISF/suricata/commit/47d6c3a3ab632dca72e60b753afa60521f0ea079)
> > 
> > Interesting, so which commit would I choose when doing the cherry-pick -x? or is that not an option in this case?
> 
> I think that the one that I indicated in the message should work, as that's the hash of the merged one :)

Yeah, I see now. I misunderstood the comment. Will get this fixed up.

**Comment by jmtaylor90** [comment]
continued in https://github.com/OISF/suricata/pull/11255

**Pull request closed without merging**